### PR TITLE
Personal number field using the textfield component

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

Using the textfield component in the personal number component so there is no duplication of the input animations.

improved the storybook for the textfield component. Moved it into the input section in storybook. 

## Justify why they are needed

only one implementation of the inputs animations. laying the groundwork for future input work.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/components/PersonalNumberField/PersonalNumberField.tsx [hunk 1]
```diff
@@ -1,25 +1,17 @@
-import { useTheme } from '@emotion/react'
-import styled from '@emotion/styled'
-import { motion } from 'framer-motion'
 import Personnummer from 'personnummer'
 import { ChangeEventHandler, InputHTMLAttributes, useState } from 'react'
+import { TextField } from '../TextField/TextField'
```
Comment (robinandeer) [inline apps/store/src/components/PersonalNumberField/PersonalNumberField.tsx:3]: nit: import from `@/components`

File: apps/store/src/components/TextField/TextField.stories.tsx [hunk 2]
```diff
@@ -1,21 +1,29 @@
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
 import { ComponentMeta, ComponentStory } from '@storybook/react'
 import { TextField } from './TextField'
 
 export default {
-  title: 'TextField',
+  title: 'Input/TextField',
```
Comment (robinandeer) [inline apps/store/src/components/TextField/TextField.stories.tsx:6]: I think there's another section called "Inputs"

**Review by robinandeer — APPROVED** [review]
Very nice :)

File: apps/store/src/components/TextField/TextField.stories.tsx (hunk 2 above)
Comment (KarnellSchultz) [reply apps/store/src/components/TextField/TextField.stories.tsx:6]: At the moment it's called 'input' but going to change it to 'inputs' rn

Screenshot 2022-12-02 at 10 56 44

**Pull request merged**